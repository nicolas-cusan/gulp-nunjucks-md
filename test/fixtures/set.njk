{% set title = 'Title via set' %}
{% extends 'parent.njk' %}
