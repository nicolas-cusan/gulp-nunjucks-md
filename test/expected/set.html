Title via set
Parent contents